syscall.go

```diff
@@ -111,6 +114,62 @@ func unmountShareDir() error {
 	return os.RemoveAll(containerMountDest)
 }
 
+func waitForBlockDevice(deviceName string) (err error) {
+	devicePath := filepath.Join(devPath, deviceName)
+
+	if _, err := os.Stat(devicePath); err == nil {
```

Comment:
Yes, successful stat on the device would mean the device is ready. I went with the route of udev events as I find it a much cleaner approach, rather than polling continuously.